dcat/index.html

```diff
@@ -45,32 +45,32 @@
+    <p style="text-indent: 100px;">The DCAT vocabulary for this version, Version 2, is available <a href="https://w3c.github.io/dxwg/dcat/rdf/dcat.ttl">here</a>.</p>
 </section>
 
 <section id="sotd">
 
     <p>
-        Since the <a href="https://www.w3.org/TR/2018/WD-vocab-dcat-2-20180508/">First Public Working Draft</a>, the main changes to the DCAT vocabulary have been:</p>
+        Since the <a href="https://www.w3.org/TR/2018/WD-vocab-dcat-2-20180508/">First Public Working Draft</a>, the main changes to the DCAT vocabulary in Version 2 have been:</p>
     <ul>
-        <li>addition of a <a href="#Class:Resource"><code>dcat:Resource</code></a> class for representing any asset than can be included in the catalog, this is
+        <li>addition of a <a href="#Class:Resource"><code>dcat:Resource</code></a> class for representing any asset than can be included in the catalog.  This is
             now the super-class of <a href="#Class:Dataset"><code>dcat:Dataset</code></a></li>
-        <li>addition of <a href="#Class:Data_Service"><code>dcat:DataService</code></a>, as a sub-class of <a href="#Class:Resource"><code>dcat:Resource</code></a>, to support cataloguing service end-points providing access to data assets</li>
-        <li>addition of ways to represent <a href="#bag-of-files">loosely structured catalogs</a>, where there is no distinction between a dataset and its distributions</li>
-        <li>more details for the ways of representing <a href="#examples-dataset-provenance">dataset provenance</a> and <a href="#quality-information">quality</a></li>
-        <li>an <a href="#dcat-sdo">alignment</a> between the DCAT vocabulary and the schema.org vocabulary</li>
+        <li>addition of <a href="#Class:Data_Service"><code>dcat:DataService</code></a>, as a sub-class of <a href="#Class:Resource"><code>dcat:Resource</code></a>, so that catalogs can now hold information about service end-points that provide access to data assets</li>
+        <li>addition of ways to represent <a href="#bag-of-files"></a>, where there is no distinction between a dataset and its distributions, such as in many open data repositories</li>
+        <li>more details for the ways of representing <a href="#examples-dataset-provenance"></a> and <a href="#quality-information"></a></li>
+        <li>an <a href="#dcat-sdo"></a> between the DCAT vocabulary and the schema.org vocabulary</li>
     </ul>
 
 
     <p>
-        The detailed differences between the two documents can be seen <a href="https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20180508%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20181011%2F">here</a>.
+        The detailed differences between Version 2 and Version 1 can be seen <a href="https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20180508%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20181011%2F">here</a>.
```

Comment:
If we want to show the difference between dcat 2 and 1 then the link should be updated, we can't put the very last dcat2 dated URI as we haven't yet voted for the CR, but in the meanwhile,  we can use  "https://w3c.github.io/dxwg/dcat/" which is a good approximation, what do you think?
```suggestion
        The detailed differences between Version 2 and Version 1 can be seen <a href="https://services.w3.org/htmldiff?doc1=https://www.w3.org/TR/vocab-dcat/&doc2=https://w3c.github.io/dxwg/dcat/">here</a>.
```

Reply:
Or maybe we can revise the text as follows:

> The detailed differences between [[VOCAB-DCAT-20140116]] and and this version of DCAT